project_setup/README.md

```diff
@@ -1,5 +1,15 @@
 # Project Setup #
 
+We recommend you use a skeleton for all new repositories following the
+directions below.
+
+For repositories from skeletons, run `pre-commit install` to run linting
+and other tools before committing a faux pas to the repository which
+will immediately cause linting failures.
+
+Once you've set up a repository, make sure to enable
+[branch protection](branch-protection.md).
+
 ## Skeleton Tool 💀🛠 ##
```

Comment:
Sentence case for headers.

Reply:
Thanks - I'm gone back and checking the existing files and headers for sentence case compliance.

Reply:
Fixed in https://github.com/cisagov/development-guide/pull/44/commits/f4bbeece384fc59b14ab45a60999c2417ea0d1b1, gotta catch em all!